src/test/unit/platform.h

```diff
@@ -35,7 +35,9 @@
 #define MAX_PROFILE_COUNT 3
 #define USE_MAG
 #define USE_BARO
+#ifndef USE_GPS
```

Comment:
These are not needed since the value of the define is not changed, and are normally omitted.

Reply:
Some unit tests rely on the definition of USE_GPS in this file, but at the same time, I have to define at compile time USE_GPS for osd.c
So either I added the ifndef here, or I had to update the makefile for all the rest of GPS related unit tests. I went with the less intrusive option.

Reply:
Put it in src/test/Makefile under the appropriate unit test xxx_test_DEFINES section.

Reply:
Just as a reminder, note that this was not needed before this PR because USE_GPS/featureIsEnabled were not used for MAX_SPEED and MAX_DISTANCE

Reply:
Yes, I already put the USE_GPS in the osd_test_DEFINES. My point is that in order to remove the ifndef, I'll have to update the rest of unit tests. I can do it, no problem. But just wanted to make sure this was the preferred way.

Reply:
Hold on. I see the problem.

Reply:
See https://github.com/betaflight/betaflight/pull/7043#discussion_r231862205.